ModuleChangelog.md

```diff
@@ -1,3 +1,36 @@
+## 2.0.0
+
+Release Date: September 16, 2021
+
+#### RELEASE NOTES
+
+```
+* Added API Key form and parameter to allow for "Auto-Binding" the user during Migration
+    * If selected, both the API Key and Connect Key values must be entered
+    * At the end of migration the ADMU will search the JumpCloud Organization for the username entered and bind the user to the system
```

Comment:
If it doesn't exist, a warning log message will display, the profile conversion will complete. The GUI should validate that the user exists in the org before migration.